.eslintrc

```diff
@@ -1,11 +1,16 @@
 {
   "parserOptions": {
-    "ecmaVersion": 2018
+    "ecmaVersion": 6
   },
-  "extends": ["eslint:recommended", "eslint-config-prettier"],
+  "extends": [
+    "eslint:recommended",
+    "eslint-config-prettier",
+    "plugin:@typescript-eslint/recommended"
+  ],
   "rules": {
     "no-unused-vars": "off",
```

Comment:
👀 Leaving unused `var`s in your code is not a good idea unless you really have to. I suggest removing this.

Reply:
Ah Yeah, this was old config.